src/db_manager.py

```diff
@@ -140,6 +140,26 @@ def add_activity(self, activity):
 
             return Result.Ok()
 
+    def get_all_activities(self, username):
```

Comment:
set the argument to user (User) instead of username just to keep the db manager interface consitent